README.markdown

```diff
@@ -142,6 +142,15 @@ Non-goals:
 Documentation is currently maintained on a [dedicated ReadTheDocs
 page](https://tinygo.readthedocs.io/en/latest/).
 
+## Slack channel
```

Comment:
I wonder whether "Slack channel" is a good name? Perhaps something like "Getting help" is better, as it is more general - maybe we'll add a forum in the future (or subreddit, or something like that).
I have looked around a bit but such a section is surprisingly rare in a README, so not sure what other projects call it.

Reply:
Excellent.  I was struggling with a section name to use.  "Getting help" does sound better, and leaves room for future expansion. :smile: